README.md

```diff
@@ -1,41 +1,143 @@
 # Eno
 
-## What is Eno?
+Compose Kubernetes deployments.
 
-Eno is a new take on configuration management for Kubernetes.
+- 🎹 **Synthesize**: generate manifests dynamically in short-lived pods
+- ♻️ **Reconcile**: apply the generated configurations to one (or more!) clusters
```

Comment:
This is technically not yet true lol, maybe local / remote?

Reply:
The current eno-reconciler composition label selectors kind of deliver on this right?

Reply:
I was thinking about the mixed targets scenario, but this calls out "configurations" so you're right